### ISSUE TITLE
Modify websiteURL for all output geojsons

### ISSUE THREAD

**Issue by Wiscmapper** [WIStCart/geodata-processing#25]
TBD for now.  Inserting this as a placeholder.

**Comment by HaydenElza** [comment]
If a feature look something like this:

~~~ json
{
    "type": "Feature",
    "properties": {
    "title": "210536",
    "label": "210536",
    "downloadUrl": "https://bin.ssec.wisc.edu/pub/wisconsinview/lidar/Adams/Adams_2010_County_Delivery/Bare_Earth_Points/LAS/210536.las",
    "websiteUrl": "https://bin.ssec.wisc.edu/pub/wisconsinview/lidar/Adams/Adams_2010_County_Delivery/Bare_Earth_Points/LAS"
    },
    "geometry": {}
}
~~~

Does that mean we would replace the basename `https://bin.ssec.wisc.edu/pub/wisconsinview/lidar/Adams/Adams_2010_County_Delivery/Bare_Earth_Points/LAS/` with something like `https://geodata.wisc.edu/data/Adams_2010_County_Delivery/Bare_Earth_Points/`?

We could add an option to specify if you want to edit the `downloadUrl`, the `websiteUrl`, or both.

**Comment by Wiscmapper** [comment]
We've been going with the idea that we would use https://www.sco.wisc.edu/data/elevationlidar/ as the websiteUrl for all Wisconsin LiDAR data, unless there was a specific reason to use something else on a case-by-case basis.  The websiteUrl is conceptually designed to be a place for somebody to go to learn more about the thing referenced.

**Comment by HaydenElza** [comment]
Addressed in d9e8643b32217207993f3b2c0024ac4e4ac2e9b0 with the addition of `update_url.py`.
